Week3/homework/step4-bonus.js

```diff
@@ -2,9 +2,10 @@
 
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
-// Add your function here. Try and come up with a good name for this function
+function uniqueValues() {
+  let newArr = Array.from(new Set(values));
+  return newArr;
+}
 
-// Replace `yourFunction` with the name of the function you just created
-const uniqueValues = yourFunction(values);
-
-console.log(uniqueValues);
+/* Second way */
+let uniqueValues = [... new Set(values)];
```

Comment:
This looks like google work. Can you explain how it works?

Because you have used the same name for the function and the variable, I can not test it without modifying your code. That puts the burden on the reviewer (me). Make sure that I can run your code without modification.